exalearn-ui/src/styles/mixins/media-query.scss

```diff
@@ -0,0 +1,51 @@
+@import '../screen-sizes.scss';
+
+@mixin media($minmax, $media) {
+	@each $screen-sizes, $resolution in $screen-sizes {
+		@if $media == $screen-sizes {
+			@media only screen and (#{$minmax}-width: $resolution) {
+				@content;
+			}
+		}
+	}
+}
+
+/*
+    How to use it?
```

Comment:
Cosmetic: It's not really ReadMe.md, so we don't need examples and really long comments, please, make short important notes above mixin